ldk/csharp/OliveHelpsLDK/OliveHelpsLDK/Whispers/Disambiguation.cs

```diff
@@ -0,0 +1,54 @@
+namespace OliveHelpsLDK.Whispers.Disambiguation
+{
+    /// <summary>
+    /// Base class for Whisper Disambiguation responses.
+    /// </summary>
+    public interface IWhisperDisambiguationResponse
+    {
+    }
+
+    /// <summary>
+    /// Represents a selection of an item
+    /// </summary>
+    public struct DisambiguationResponse : IWhisperDisambiguationResponse
```

Comment:
I'm thinking that whenever the namespace includes the relevant noun that it's perhaps redundant to include the noun again in the object name.

So this becomes `OliveHelpsLDK.Whispers.Disambiguation.Response`. 

@RusGraf what do you think?

Reply:
And it's entirely possible I have not been consistent/following this myself :)

Reply:
I would think you are correct if the situation be like:
```
public class WhisperDisambiguation 
{
    public Response {get; set;} // not public WhisperDisambiguationResponse {get; set;}
}
```

But it is totally correct to have Foo : IFoo. To be honest I don't mind long class or struct names. As long as they describing the intent.

Reply:
Ok, I can update those names